hicommonwealth/commonwealth
# Magic login for cosmos

## Link to Issue
Closes: #3824 

## Description of Changes
- Adds Magic login for Cosmos SDK chains

## Test Plan
- CA (click around) tested on local:
  - using https://tempmail.email/ to generate temporary email addresses...
  - **Use magic link to log in every time**:
      - Log in to /osmosis -> log out -> go to /dydx -> log in with same email address
      - Log out -> Log in to /dydx with new email address-> log out -> go to /osmosis -> log in with same email address 
      - Join other Cosmos and non-Cosmos communities in both cases -> expect successful subscriptions.
      - Expect previous community subscriptions to be persisted.

## Deployment Plan
- update env var `MAGIC_SUPPORTED_BASES=substrate,ethereum,cosmos`

## Other Considerations

-
